python/src/eiger_detector/control/eiger_options.py

```diff
@@ -1,26 +1,19 @@
 class EigerOption(object):
 
     def __init__(self, options):
-
         self.options = options
 
     def get_allowed_values(self):
-
-        return list(map(str, range(0, len(self.options))))
+        return list(map(str, range(len(self.options))))
 
     def get_option(self, idx):
-
         return self.options[int(idx)]
 
     def get_index(self, val):
-
         return str(self.options.index(val))
 
-trigger_options = EigerOption(['ints', 'inte', 'exts', 'exte'])
-compression_options = EigerOption(['lz4', 'bslz4'])
-header_options = EigerOption(['all', 'basic', 'none'])
-mode_options = EigerOption(['disabled', 'enabled'])
-
-option_config_items = ['compression', 'trigger_mode', 'header_detail', 'mode']
 
-option_config_options = {'compression': compression_options, 'trigger_mode': trigger_options, 'header_detail': header_options, 'mode': mode_options}
+option_config_options = {'compression': EigerOption(['lz4', 'bslz4']),
```

Comment:
This is nicer, but now it's being used directly, the name of a bit unwieldy. Maybe eiger_config_options?

Reply:
Good point. I will need to have a second look at these parameters anyway if we want this adapter to report to FastCS - there's some disparity between the parameter type as reported by the parameter tree (str) whereas they expect an integer value to be written in, will avoid having to put any extra ugly logic into odin-fastcs to generate nicer dropdowns.

Reply:
We don't want this adapter to report to FastCS. eiger-fastcs talks directly to the detector, so this adapter will be redundant.